### PR TITLE
fix: My non-carbonless NFT is carbonless

### LINKED ISSUE #5310: My non-carbonless NFT is carbonless 🌋🌱
### What happened?

Basilisk minting: I added NFT to my collection and I toggled off "add KodaForest attribute"  (I don't want to pay 1 USD for it to be carbonless)

image

I minted it and [my NFT](https://kodadot.xyz/bsx/gallery/1748272741-1?page=1 ) still has a "carbonless" property. This cannot be right.
Screenshot 2023-03-20 at 19 23 39

### Please reproduce in steps

Basilisk, Create, NFT, toggle off "add KodaForest attribute"

### Expected Behavior

- Not to add the "carbonless" property to NFT if I decided to toggle that option off during minting.
- Maybe hide the option KodaForest if someone opens on-chain attributes when a person toggles off the option
- Review please - am I paying an extra 1 USD fee for KodaForest if I toggle this option ON?? 

### What browsers are you seeing the problem on?

Chrome

### At which address did you encounter bug?

kodadot.xyz

### Are you logged in?

Yes

### Which wallet you are using?

PolkadotJS

### Screenshots

added

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5310
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

tested on /bsx/gallery/290618266-2

image

### CODE REVIEW

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

**Comment by Jarsen136** [comment]
✅  remove carbonless from BSX

File: components/bsx/Create/CreateToken.vue [hunk 1]
```diff
@@ -171,7 +171,11 @@ export default class CreateToken extends mixins(
   }
 
   get hasCarbonOffset() {
-    return this.preferencesStore.hasCarbonOffset
+    return this.preferencesStore.getHasCarbonOffset
+  }
+
+  set hasCarbonOffset(value: boolean) {
+    this.preferencesStore.setHasCarbonOffset(value)
```
[the remark this answers is not in the document]
Comment (Jarsen136) [reply components/bsx/Create/CreateToken.vue]: done

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 15 usd

**Pull request merged**